### PR TITLE
Fix ai binding tests

### PR DESCRIPTION
**Pull request by G4brym**
## What this PR solves / how to test

The next workerd version, includes some changes in the ai binding, this pr fixes the tests that will start failing

## Author has addressed the following

- Tests
  - [ ] TODO (before merge)
  - [x] Included
  - [ ] Not necessary because:
- Changeset ([Changeset guidelines](https://github.com/cloudflare/workers-sdk/blob/main/CONTRIBUTING.md#changesets))
  - [ ] TODO (before merge)
  - [ ] Included
  - [x] Not necessary because:
- Public documentation
  - [ ] TODO (before merge)
  - [ ] Cloudflare docs PR(s): <https://github.com/cloudflare/cloudflare-docs/pull/>...
  - [x] Not necessary because:

### CODE REVIEW

**Comment by G4brym** [comment]
@Cherry a new workerd release would take a few days, and block other features
I think its better to remove the test that is triggering the build to fail currently

**Comment by Cherry** [comment]
Whoops, it seems I missed the removal of `logs` when mentioning the breaking change that removing `getLogs` would have been in https://github.com/cloudflare/workerd/pull/2044.

Perhaps skipping the test here temporarily (`it.skip`) and then restoring it after `workerd` is patched to revert the breaking change would be a better approach, instead of removing the `binding` test entirely? It's a good thing this was caught here - and it would be good to catch other breaking changes like this in the future too instead of removing it entirely.

**Comment by G4brym** [comment]
Agree, i've just openned a pr in workerd to re add it
https://github.com/cloudflare/workerd/pull/2095/files

File: fixtures/ai-app/tests/index.test.ts [hunk 1]
```diff
@@ -20,18 +20,21 @@ describe("'wrangler dev' correctly renders pages", () => {
...
-				logs: [],
-				options: {},
-			},
-			fetch: "function",
-			run: "function",
+		expect((content as Record<string, object>).fetch).toEqual("function");
+		expect((content as Record<string, object>).run).toEqual("function");
+	});
+
+	it.skip("ai binding properties", async ({ expect }) => {
```
Comment (RamIdeas) [inline fixtures/ai-app/tests/index.test.ts:30]: Would be good to comment when this test can be unskipped

```suggestion
	// TODO: unskip when https://github.com/cloudflare/workerd/pull/2095 is merged and released
	it.skip("ai binding properties", async ({ expect }) => {
```

**Review by RamIdeas — APPROVED** [review]

**Pull request merged**